docs/rules/lines-around-comment.md

```diff
@@ -48,123 +46,114 @@ Empty lines are also not required at the beginning or end of a file.
 
 ## Options
 
-This rule has 10 options.
+This rule has an object option:
 
-2 options for block comments:
+* `"beforeBlockComment": true` (default) enforces an empty line before block comments
+* `"afterBlockComment": true` enforces an empty line after block comments
+* `"beforeLineComment": true` enforces an empty line before line comments
+* `"afterLineComment": true` enforces an empty line after line comments
+* `"allowBlockStart": true` allows comments to be present at the start of block statements
```

Comment:
Does rewording it to `to appear` instead of `to be present` help? I changed them to that for the time being, but we can discuss this further.